src/metabase/search/appdb/index.clj

```diff
@@ -247,17 +248,14 @@
 
 (defn- document->entry [entity]
   (-> entity
-      (select-keys
-       ;; remove attrs that get explicitly aliased below
-       (remove #{:id :created_at :updated_at :native_query}
-               (conj search.spec/attr-columns :model :display_data :legacy_input)))
+      (select-keys (conj search.spec/attr-columns :model :display_data :legacy_input))
```

Comment:
This bit is exactly the same functionally, I just found the existing impl to be unclear since we were removing some keys to rename them, and removing others entirely - feels simpler to explicitly rename what we want to rename and remove the one (`:native_query`) that we want to remove.